dvc/utils/compat.py

```diff
@@ -28,6 +28,12 @@ def encode(u, encoding=None):
     return u.encode(encoding, "replace")
 
 
+def decode(u, encoding=None):
```

Comment:
Let's use cast_unicode_py2 from https://github.com/ipython/ipython_genutils/blob/master/ipython_genutils/py3compat.py . We've been using some snippets from that down below and above (e.g. `def encode` above).

Reply:
@efiop , do you mean `decode`? `cast_unicode_py2` doesn't make a lot of sense because `type('\xe8')` is `str`:

```python
def cast_unicode(s, encoding=None):
    if isinstance(s, bytes):
        return decode(s, encoding)
    return s
```

Reply:
`str` is the same as `bytes` on py2 though, by default.

Reply:
TIL! that's great